app/search/styles.scss

```diff
@@ -2,10 +2,19 @@
     background-color: $color-bg-gray;
 }
 
+[data-analytics-scope='Search page main'] {
+    margin-right: 3rem;
+    width: 70%;
+}
+
 .heading-wrapper {
     text-align: center;
     color: $color-text-white;
     background-color: $osf-dark-blue-navbar;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
```

Comment:
This, 'text-align: center', and 'flex-direction: row' should be defaults and not may not need to be explicitly stated, but seem like good practice. They can be removed if truly unnecessary.